src/components/Calculator.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+const CalculatorUI = () => (
+  <div className="calculator">
+    <div className="screen">
+      <p>0</p>
+    </div>
+    <div className="keys">
+      <button type="button">AC</button>
+      <button type="button">+/-</button>
+      <button type="button">%</button>
+      <button type="button" className="operation">÷</button>
+    </div>
+    <div className="keys">
+      <button type="button">7</button>
+      <button type="button">8</button>
+      <button type="button">9</button>
+      <button type="button" className="operation">x</button>
+    </div>
+    <div className="keys">
+      <button type="button">6</button>
+      <button type="button">5</button>
+      <button type="button">4</button>
+      <button type="button" className="operation">-</button>
+    </div>
+    <div className="keys">
+      <button type="button">3</button>
+      <button type="button">2</button>
+      <button type="button">1</button>
+      <button type="button" className="operation">+</button>
+    </div>
+    <div className="keys">
+      <button type="submit" className="zero">0</button>
+      <button type="button" className="dot">.</button>
+      <button type="button" className="operation">=</button>
+    </div>
+  </div>
```

Comment:
- Great job for implementing the given design 👍 however you have finished all functionality in one component which is not the best practice 👌 kindly separate it into a different component and use it in `CalculatorUI` to follow single responsibility, and full fill the requirement👍

Reply:
Thank you very much for your feedback. However, this is not part of the project requirements as specified by Microverse @binhussen